workflows/export/vx_export_vod.go

```diff
@@ -135,15 +135,47 @@ type vxExportVodService struct {
 	errs                   []error
 }
 
-func prepareAudioFiles(ctx workflow.Context, mergeResult MergeExportDataResult, tempDir paths.Path) (map[string]paths.Path, error) {
+func prepareAudioFiles(ctx workflow.Context, mergeResult MergeExportDataResult, tempDir paths.Path, normalizeAudio bool) (map[string]paths.Path, error) {
 	prepareFilesSelector := workflow.NewSelector(ctx)
 
+	if normalizeAudio {
+		langs, err := wfutils.GetMapKeysSafely(ctx, mergeResult.AudioFiles)
+		if err != nil {
+			return nil, err
+		}
+		normalizedFutures := map[string]workflow.Future{}
+		// Normalize audio
+		for _, lang := range langs {
+			audio := mergeResult.AudioFiles[lang]
+			ctx = workflow.WithChildOptions(ctx, wfutils.GetDefaultWorkflowOptions())
+			future := wfutils.ExecuteWithQueue(ctx, activities.NormalizeAudioActivity, activities.NormalizeAudioParams{
+				FilePath:              audio,
+				TargetLUFS:            -24,
```

Comment:
wasnt it -15/-14 you found

Reply:
Then I need to put in a switch otherwise all VODs will also be on that level. Currently everything else in the BCCM app is @-24...